### ISSUE TITLE
HDMI Audio not working

### ISSUE THREAD

**Issue by adityahota** [LuletterSoul/Dell-XPS-15-9570-macOS-Mojave#63]
I was attempting to figure out why audio does not work over HDMI--AppleALC, WhateverGreen, and Lilu are all installed, but when connecting an HDMI or DisplayPort (via USB-C port), the audio selector only says internal speakers.

Does anyone else have this issue?

**Comment by yongjiajun** [comment]
Yes I can confirm this, my TV shows up as an audio source on my MacBook but not on my XPS 15 with the latest release (v1.5) of EFI..

**Comment by LuletterSoul** [comment]
@adityahota 
Hi, you can see more discussion at bavariancake/XPS9570-macOS#42

**Comment by adityahota** [comment]
Thanks! I actually have it working using the fix in xxxzc’s repo.
